mappings/net/minecraft/client/render/RenderPhase.mapping

```diff
@@ -70,15 +129,38 @@ CLASS net/minecraft/class_4668 net/minecraft/client/render/RenderPhase
 			ARG 2 y
 	CLASS class_4683 Texture
 		FIELD field_21397 id Ljava/util/Optional;
-		FIELD field_21398 bilinear Z
+		FIELD field_21398 blur Z
 		FIELD field_21399 mipmap Z
+		METHOD <init> (Lnet/minecraft/class_2960;ZZ)V
+			ARG 1 id
+			ARG 2 blur
+			ARG 3 mipmap
 	CLASS class_4684 Texturing
 	CLASS class_4685 Transparency
 	CLASS class_4686 WriteMaskState
-		FIELD field_21400 color Z
-		FIELD field_21401 depth Z
+		FIELD field_21400 writeColor Z
+		FIELD field_21401 writeDepth Z
 		METHOD <init> (ZZ)V
 			ARG 1 color
 			ARG 2 depth
-	CLASS class_5939
+	CLASS class_5939 TextureBase
+		METHOD <init> (Ljava/lang/Runnable;Ljava/lang/Runnable;)V
+			ARG 1 apply
+			ARG 2 unapply
 		METHOD method_23564 getId ()Ljava/util/Optional;
+	CLASS class_5940 Textures
+		FIELD field_29453 id Ljava/util/Optional;
+		METHOD <init> (Lcom/google/common/collect/ImmutableList;)V
+			ARG 1 textures
+		METHOD method_34560 create ()Lnet/minecraft/class_4668$class_5940$class_5941;
+		CLASS class_5941 Builder
+			FIELD field_29454 textures Lcom/google/common/collect/ImmutableList$Builder;
+			METHOD method_34562 build ()Lnet/minecraft/class_4668$class_5940;
+			METHOD method_34563 add (Lnet/minecraft/class_2960;ZZ)Lnet/minecraft/class_4668$class_5940$class_5941;
+				ARG 1 id
+				ARG 2 blur
+				ARG 3 mipmap
+	CLASS class_5942 Shader
+		FIELD field_29455 shaderSupplier Ljava/util/Optional;
```

Comment:
Can this be just called `supplier`?

Reply:
I would kind of rather it reflect what's _in it_ though, no?

Reply:
It's a supplier that provides a shader, hence `shaderSupplier`

Reply:
It's already in a class named `RenderPhase.Shader`, so it is redundant to also name the field `shaderSupplier`.

Reply:
```suggestion
		FIELD field_29455 supplier Ljava/util/Optional;
```